hibernate-core/src/main/java/org/hibernate/dialect/H2Dialect.java

```diff
@@ -138,16 +133,10 @@ public H2Dialect(DatabaseVersion version) {
 		// 1.4.200 introduced changes in current_time and current_timestamp
 		useLocalTime = version.isSameOrAfter( 1, 4, 200 );
 
-		if ( version.isSameOrAfter( 1, 4, 32 ) ) {
-			this.sequenceInformationExtractor = version.isSameOrAfter( 1, 4, 201 )
-					? SequenceInformationExtractorLegacyImpl.INSTANCE
-					: SequenceInformationExtractorH2DatabaseImpl.INSTANCE;
-			this.querySequenceString = "select * from INFORMATION_SCHEMA.SEQUENCES";
-		}
-		else {
-			this.sequenceInformationExtractor = SequenceInformationExtractorNoOpImpl.INSTANCE;
-			this.querySequenceString = null;
-		}
+		this.sequenceInformationExtractor = version.isSameOrAfter( 1, 4, 201 )
+				? SequenceInformationExtractorLegacyImpl.INSTANCE
+				: SequenceInformationExtractorH2DatabaseImpl.INSTANCE;
+		this.querySequenceString = "select * from INFORMATION_SCHEMA.SEQUENCES";
```

Comment:
Please remove the field and simply return `"select * from INFORMATION_SCHEMA.SEQUENCES"` from the `getQuerySequencesString` method.

Reply:
Same comment as above